### PR TITLE
lib/sandbox: clarify variable names

### PR DESCRIPTION
**Pull request by vbatts**
three uses of `ns` :-\

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: lib/sandbox/sandbox.go [hunk 1]
```diff
@@ -23,13 +23,13 @@ import (
 // NetNs handles data pertaining a network namespace
 type NetNs struct {
 	sync.Mutex
-	ns       ns.NetNS
+	nn       ns.NetNS
```
Comment (wking) [inline lib/sandbox/sandbox.go]: I'd rather use `netNS` or some such here to reduce the risk of future collisions.

Comment (rhatdan) [reply lib/sandbox/sandbox.go]: Yes I like netNS

Comment (vbatts) [reply lib/sandbox/sandbox.go]: alrighty

**Comment by vbatts** [comment]
updated. PTAL

File: lib/sandbox/sandbox.go [hunk 2]
```diff
@@ -23,13 +23,13 @@ import (
...
 	sync.Mutex
-	ns       ns.NetNS
+	netNS    ns.NetNS
 	symlink  *os.File
 	closed   bool
 	restored bool
 }
 
-func (ns *NetNs) symlinkCreate(name string) error {
+func (nns *NetNs) symlinkCreate(name string) error {
```
Comment (rhatdan) [inline lib/sandbox/sandbox.go]: Can't we call this netns?

Comment (vbatts) [reply lib/sandbox/sandbox.go]: it needs to be public, since it's used outside this package, right?

Comment (vbatts) [reply lib/sandbox/sandbox.go]: unless you mean the internal reference of the function?

Comment (rhatdan) [reply lib/sandbox/sandbox.go]: Yes the internal name.

Comment (vbatts) [reply lib/sandbox/sandbox.go]: done

**Comment by vbatts** [comment]
updated. PTAL.

**Comment by rhatdan** [comment]
LGTM
/test all

**Comment by vbatts** [comment]
rebased. PTAL

**Comment by vbatts** [comment]
all green. PTAL

**Comment by baude** [comment]
lgtm

**Comment by mheon** [comment]
LGTM

**Comment by rhatdan** [comment]
Merging.

**Pull request merged**